.github/workflows/component-builder.yml

```
@@ -23,40 +28,48 @@ jobs:
-        working-directory: ./containers/fedora
         env:
           PYTHONPATH: ${{ github.workspace }}
           NO_SECRETS: "true"
         run: ./build.sh
-      - name: Save container image as tarball
-        env:
-          local_repository: "localhost/fedora"
-          remote_repository: "quay.io/openshift-cnv/qe-cnv-tests-fedora-staging"
-          arch_tag: "${{ env.FEDORA_VERSION }}-${{ env.CPU_ARCH }}"
+      - name: Test upload to quay.io
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC1083:warning:1:45: This { is literal. Check expression (missing ;/\n?) or quote it

(shellcheck)